llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -3164,6 +3164,18 @@ def : GCNPat <
                         (as_i1timm $bound_ctrl))
 >;
 
+class SMPrefetchGetPcPat<string type, int cache_type> : GCNPat <
```

Comment:
This pattern also interprets the "address" argument as being an offset from PC, so it should also be removed from this version of the patch.